### ISSUE TITLE
A11y_Model Builder_Evaluate_Try Your Model_Use Of Color : When the "Horizon" field is not set, an error is shown with the help of color(red) only.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2189]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Visual Studio Client; #SH-ModelBuilder-Win32-Jun2022; #Win32; #WCAG1.4.1; #Use Of Color;

### Environment Details:
Application Name: Model Builder
Windows Version: Windows 11 22H2 OS Builder 25145.1011
Visual Studio: Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview Version 17.3.0 Preview 2.0

### Repro Steps:

1. Install Visual studio and open Visual studio
2. Navigate to C# project name and right click on it
3. Navigate to "Add" button and press enter
4. Navigate to "Machine learning model" and press enter
5. "Add new item" popup will open
6. Navigate to "Add" button and press enter
7. "Select scenario" popup will open and select "Forecasting" scenario
8. "Select training environment" page will open then select "Next step"
9. Tab till "Evaluate" and press enter.
10. tab till Try Your Model section.
11. Tab till "Horizon" field keeping the field empty, tab to next control.
12. Check whether When the "Horizon" field is not set, an error is shown with the help of color(red) only or not.

### Actual Result:
When the "Horizon" field is not set, an error is shown with the help of color(red) only.

### Expected Result:
When the "Horizon" field is not set, an error should be shown with some text message or symbol not with the help of color(red) only.

![horizon](https://user-images.githubusercontent.com/91586101/177323142-c215adbe-3703-44d2-91b6-ecb66ca32791.gif)

**Comment by beccamc** [comment]
@LittleLittleCloud How is the red border even getting set here? I don't see it in the Forecasting xaml https://github.com/dotnet/machinelearning-tools/blob/428b58214eaacf5c6bfeec45f789ec22e7bc8b3a/src/Microsoft.ML.ModelBuilder/ToolWindows/ForecastingTryOutControl.xaml#L29

**Comment by beccamc** [comment]
I think we could remove the red border and just disable the predict button until there is input. That is what we do for train time, horizon, and other input boxes.

**Comment by beccamc** [comment]
@manaswita-chougule  Please verify this on build 16.13.10.2241902 available from the Marketplace https://marketplace.visualstudio.com/items?itemName=MLNET.ModelBuilder2022

**Comment by manaswita-chougule** [comment]
Verified the issue in below environment:
Microsoft Visual Studio Enterprise 2022 (64-bit) - Preview
Version 17.4.0 Preview 1.0

"Horizon" control get disabled if it's not set to any value, hence the issue is resolved. 
Closing the bug. Please find below attachment for reference.
![2189_use of color_fixed](https://user-images.githubusercontent.com/91586101/187135039-9367f057-2384-47cc-92a9-de0c5aee6e2c.gif)
